static/styles/javascript/scripts.js

```diff
@@ -0,0 +1,98 @@
+$(document).ready(function() {
+
+    var writeMessage;
+    // building message element and appending to the list of all messages
+    writeMessage = function(message) {
+        this.text = message.text, this.side = message.side;
+        this.write = function(localthis) {
+            return function (e) {
+                var today = new Date()
+                //  time of the message sent
+                var time = today.getHours() + ":" + today.getMinutes() + ":" + today.getSeconds();
+                // ul of all messaged to which the new one will be appended
+                var messagesList = $('#chat_content').contents();
+                // building the message element with user message
+                var msgElement = '<li class="mar-btm"><div class="media-body pad-hor speech-right">'
```

Comment:
This would probably be easier written as a template string ([MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals)) - it reduces the need for all the `+`